src/advanced-settings/__mocks__/advancedSettings.js

```diff
@@ -13,4 +13,11 @@ module.exports = {
     hide_on_enabled_publisher: false,
     value: true,
   },
+  displayName: {
```

Comment:
please change naming to `camelCase` here and above

Reply:
Done, thanks